manager/assets/modext/widgets/security/modx.panel.user.group.js

```diff
@@ -313,16 +309,20 @@ Ext.extend(MODx.panel.UserGroup,MODx.FormPanel,{
         }
         var r = this.config.record;
         this.getForm().setValues(r);
-        Ext.defer(function() {
-            Ext.get('modx-user-group-header').update('<h2>'+_('user_group')+': '+Ext.util.Format.htmlEncode(r.name)+'</h2>');
-        }, 250, this);
+        Ext.getCmp('modx-user-group-header').html = Ext.util.Format.htmlEncode(r.name);
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(Ext.util.Format.htmlEncode(r.name));`